src/main/java/gregtech/api/recipes/ModHandler.java

```diff
@@ -738,4 +740,29 @@ public static ItemStack getSmeltingOutput(ItemStack input) {
         if (input.isEmpty()) return ItemStack.EMPTY;
         return OreDictUnifier.getUnificated(FurnaceRecipes.instance().getSmeltingResult(input));
     }
+
+    public static void removeSmeltingEBFMetals() {
+
+        Map<ItemStack, ItemStack> furnaceList = FurnaceRecipes.instance().getSmeltingList();
+
+        Iterator<Map.Entry<ItemStack, ItemStack>> recipeIterator = furnaceList.entrySet().iterator();
+
+        while(recipeIterator.hasNext()) {
+            Map.Entry<ItemStack, ItemStack> recipe = recipeIterator.next();
+
+            ItemStack output = recipe.getValue();
+            MaterialStack ms = OreDictUnifier.getMaterial(output);
+
+            if(ms != null) {
+                Material material = ms.material;
+                if(material.hasProperty(PropertyKey.BLAST)) {
+                    recipeIterator.remove();
+                    if(ConfigHolder.misc.debug) {
+                        GTLog.logger.info("Removing Smelting Recipe for EBF material {}", LocalizationUtils.format(material.getUnlocalizedName()));
```

Comment:
Perhaps check for input/output as validation, make sure its a dust => ingot of some kind in case another mod does a different kind of recipe that isn't meant for bypassing progression?